mne/utils/check.py

```diff
@@ -156,6 +156,8 @@ def _check_fname(fname, overwrite=False, must_exist=False, name='File',
                  need_dir=False):
     """Check for file existence, and return string of its absolute path."""
     _validate_type(fname, 'path-like', name)
+    fname = op.expanduser(fname)
```

Comment:
now the modification of `fname` happens it two places -- here and below during the `return`. Can you move the `fname = str(op.abspath(fname))` after this line, then just `return fname` at the end? That way it's immediately clear what all operations we are actually doing to `fname` before returning it.